internal/ethapi/transaction_args.go

```diff
@@ -78,32 +79,34 @@ func (args *TransactionArgs) setDefaults(ctx context.Context, b Backend) error {
 		return errors.New("both gasPrice and (maxFeePerGas or maxPriorityFeePerGas) specified")
 	}
 	// After london, default to 1559 unless gasPrice is set
-	if b.ChainConfig().IsLondon(b.CurrentBlock().Number()) && args.GasPrice == nil {
+	head := b.CurrentHeader()
+	if b.ChainConfig().IsLondon(head.Number) && args.GasPrice == nil {
 		if args.Tip == nil {
-			tip, err := b.SuggestPrice(ctx)
+			tip, err := b.SuggestGasTipCap(ctx)
 			if err != nil {
 				return err
 			}
 			args.Tip = (*hexutil.Big)(tip)
 		}
 		if args.FeeCap == nil {
-			feeCap, err := b.SuggestFeeCap(ctx)
-			if err != nil {
-				return err
-			}
+			feeCap := new(big.Int).Add((*big.Int)(args.Tip), head.BaseFee)
```

Comment:
For the post-london 1559 transactions, maybe it's better to add 2*baseFee here?

Reply:
Ah, shit, good catch, that was the intent